### PR TITLE
Challenge 14: Action Controller

### PR DESCRIPTION
**Pull request by wildmaples**

### CODE REVIEW

File: mirth.rb [hunk 1]
```diff
@@ -35,25 +33,22 @@ class DailyData < ActiveRecord::Base; end
...
-  }
-
-  match '*path', via: :all, to: -> environment {
-    request = Rack::Request.new(environment)
-    response = Rack::Response.new  
-    response.content_type = "text/plain"
-    response.write("✅ Received a #{request.request_method} request to #{request.path}!")
-    response.finish
-  }
+    redirect_to("/show/data")
```
Comment (wildmaples) [inline mirth.rb:42]: The default redirect status is 302 when there isn't a status specified. 

```
127.0.0.1 - - [23/Feb/2021:13:14:42 -0500] "POST /add/data HTTP/1.1" 302 - 0.2281
```

I figured we keep it simple here, but we can also `redirect_to("/show/data", status: 303)`

Comment (tomstuart) [reply mirth.rb:42]: I agree it’s good to keep it simple, but this is a good opportunity to use another nice feature of Action Controller. Check out [the documentation for the `:status` option](https://guides.rubyonrails.org/layouts_and_rendering.html#the-status-option) (supported by both `render` and `redirect_to`) for a new and more readable way to set the status code.

**Review by tomstuart — APPROVED** [review]
Looks good, nice work! Find a nice way to set the redirect status to `303` and ship it 🎉

File: mirth.rb (hunk 1 above)
Comment (wildmaples) [reply mirth.rb:42]: Added `render(status: :see_other)`

File: mirth.rb [hunk 2]
```diff
@@ -35,25 +33,23 @@ class DailyData < ActiveRecord::Base; end
...
+
+router = ActionDispatch::Routing::RouteSet.new
+
+router.draw do
+  get '/show/data', to: DailyDataController.action(:show_data)
+  post '/add/data', to: DailyDataController.action(:add_data)
+  match '*path', via: :all, to: DailyDataController.action(:all_paths)
 end 
 
 app_with_charset = Rack::Charset.new(router, 'utf-8')
```
Comment (tomstuart) [inline mirth.rb:55]: Sorry, I forgot to ask before you merged, but: do you still need this?

File: mirth.rb (hunk 1 above)
Comment (tomstuart) [reply mirth.rb:42]: As you can see if you run it 👀, this isn’t right.

**Pull request merged**